### DISCUSSION TITLE
An error occurred while selecting row data(Get data by custom query)

### DISCUSSION THREAD

**Discussion by zhaojing1987** [Budibase/budibase#5516] [Help]
When i show data with table，I display the data in two ways：
1. Data Provider：Data--> Queries
2.  Data Provider：Data--> Tables

An error occurred while selecting table data，With the data displayed by the first way, when I want to select a row, it actually selects all rows，it just show :“1 row selected”. But the second way is right.

![image](https://user-images.githubusercontent.com/7624828/164361313-060d73d4-2de6-4f69-b5fb-94eff22fba8c.png)

**Comment by melohagan** [comment]
Hi @zhaojing1987 

Thank you for bringing this to our attention. Another user has raised the bug here: https://github.com/Budibase/budibase/issues/5491
